cmd/offlinesync/offlinesync.go

```diff
@@ -53,13 +76,11 @@ func Run(v *viper.Viper) (int, error) {
 
 // syncOfflineActivityLegacy syncs the old offline activity by sending heartbeats
 // from the legacy offline queue to the WakaTime API.
-func syncOfflineActivityLegacy(v *viper.Viper, queueFilepath string) error {
+func syncOfflineActivityLegacy(v *viper.Viper, queueFilepath string, paramOffline params.Offline) error {
 	if queueFilepath == "" {
 		return nil
 	}
 
-	paramOffline := params.LoadOfflineParams(v)
```

Comment:
Can we keep loading params offline here?

Reply:
That's doing unnecessary work, because we've already loaded offline params in both the parent functions.

Reply:
It's using pointers so there's no overhead calling it again. It's also recommended to not pass unused params to the functions making it harder to test and guessing what's used or not.